frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -220,6 +239,18 @@ export default {
       }
       this.validateInput()
     },
+    onRemovedZone (removedZone) {
```

Comment:
Wouldn't it be better to have a sync property for the zonesNetworkConfiguration. I still do not understand why only the last zone is emitted.